src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -1467,6 +1467,9 @@ type Itinerary {
     """How far the user has to walk, in meters."""
     walkDistance: Float
 
+    """CO₂ emissions of the trip on this itinerary, in grams per person."""
+    emissions: Float
```

Comment:
We should make a wrapper type Emissions which holds co2 as a field. This field should be called something like `emissionsPerPerson`.  I think @t2gran suggested at some point that we should have some common scalar for emissions but we weren't sure how much the units differ between different emission types. Would it make any sense to make a scalar like Grams?

Reply:
I agree. A single value is too simplistic and not future-proof. I can very well forsee someone wanting to calculate NOx emissions.

Reply:
Se my comment on the Leg/Itinerary - the same thing apply to this. The types should be `Emission` and `Gram`(singular). The context `perPersion` is probably a field name for `Emission`.

Reply:
Added Emissions class. The desired structure is still a little unclear to me.